docs/sources/source/nlp/dialog_state_tracking_trade.rst

```diff
@@ -32,15 +32,15 @@ multiple turns, possibly spanning over several different domains.
 The MultiWOZ Dataset
 --------------------
 
-The Multi-Domain Wizard-of-Oz dataset (`MultiWOZ`_) is a collection of human-to-human conversations spanning over \
+The Multi-Domain Wizard-of-OZ dataset (`MultiWOZ`_) is a collection of human-to-human conversations spanning over \
```

Comment:
Why **OZ** instead of **Oz**?
I took the name from Budzianowski's paper: " To address this fundamental obstacle, we introduce the Multi-Domain Wizard-of-**Oz** dataset (MultiWOZ)"

Reply:
You are right. Reverted back.